child-project/build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+dependencies {
+    implementation("org.apache.commons:commons-lang3:3.12.0")
+    testFixturesImplementation("org.apache.commons:commons-lang3:3.12.0")
```

Comment:
This seems to be a K2 change, `testFixturesImplementation` no longer extends `implementation` 🤷🏻